integrationtest/v5/order/order_test.go

```diff
@@ -0,0 +1,30 @@
+//go:build integrationtestv5order
+
+package integrationtestv5order
+
+import (
+	"testing"
+
+	"github.com/hirokisan/bybit/v2"
+	"github.com/hirokisan/bybit/v2/integrationtest/testhelper"
+	"github.com/stretchr/testify/require"
+)
+
+func TestCreateOrder(t *testing.T) {
+	client := bybit.NewTestClient().WithAuthFromEnv()
+	price := "10000.0"
+	res, err := client.V5().Order().CreateOrder(bybit.V5CreateOrderParam{
+		Category:  bybit.CategoryV5Spot,
```

Comment:
need to be upgrade to Unified Trading Account

ref: https://testnet.bybit.com/en-US/help-center/bybitHC_Article?language=en_US&id=000001872